### PR TITLE
Add Multinode job to run Tempest tests

### PR DESCRIPTION
**Pull request by rlandy**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: zuul.d/tempest_multinode.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              tenant:
+                ip: 172.19.0.5
+      cifmw_extras:
+        - '@scenarios/centos-9/ci.yml'
+        - '@scenarios/centos-9/ci-build.yml'
+        - '@scenarios/centos-9/ceph_backends.yml'
+    pre-run:
+      - ci/playbooks/e2e-prepare.yml
+    run:
+      - ci/playbooks/dump_zuul_data.yml
```
Comment (viroel) [inline zuul.d/tempest_multinode.yaml:47]: already on parent pre-run:
https://github.com/openstack-k8s-operators/ci-framework/blob/846552bfa08dc8d447ba577c3f2ec1f90c169739/zuul.d/base.yaml#L124

File: zuul.d/tempest_multinode.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      - ci/playbooks/e2e-run.yml
+    post-run:
+      - ci/playbooks/collect-logs.yml
```
Comment (viroel) [inline zuul.d/tempest_multinode.yaml:50]: already on parent post-run:
https://github.com/openstack-k8s-operators/ci-framework/blob/846552bfa08dc8d447ba577c3f2ec1f90c169739/zuul.d/base.yaml#L127

**Comment by rlandy** [comment]
/hold 
retesting

**Comment by rlandy** [comment]
Passing test: https://logserver.rdoproject.org/54/36254/279/check/cifmw-multinode-tempest/b6f9cb4/controller/ci-framework-data/tests/tempest/stestr_results.html

**Comment by son-vyas** [comment]
/lgtm 

Below are the tempest tests results which are green basically.
https://logserver.rdoproject.org/65/665/54a56cd4452f3152529cdd9d0fae7a31d0932782/github-check/podified-multinode-edpm-e2e-nobuild-tagged-crc/6b90803/controller/ci-framework-data/tests/tempest/stestr_results.html

**Review by cjeanner — COMMENTED** [review]
/approve

**Pull request merged**